zaza/openstack/charm_tests/test_utils.py

```diff
@@ -772,6 +773,8 @@ def launch_guest(self, guest_name, userdata=None, use_boot_volume=False,
         :param attach_to_external_network: Attach instance directly to external
                                            network.
         :type attach_to_external_network: bool
+        :param keystone_session: Keystone session to use.
+        :type keystone_session: keystoneauth1.session.Session
```

Comment:
The `:type ..:` is `Optional[keystoneauth1.session.Session]` isn't it?

Reply:
correct, will fix that.